static/js/components/SortPicker.js

```diff
@@ -1,30 +1,60 @@
 // @flow
 import React from "react"
-import R from "ramda"
+import { Menu, MenuItem, MenuAnchor } from "rmwc/Menu"
+
+import { DropDownArrow, DropUpArrow } from "./UserMenu"
 
 import {
   VALID_POST_SORT_LABELS,
   VALID_COMMENT_SORT_LABELS
 } from "../lib/sorting"
 
-type PostSortProps = {
+type Props = {
   updateSortParam: (t: string) => void,
   value: string
 }
 
-const SortPicker = R.curry(
-  (labels, { updateSortParam, value }: PostSortProps) => (
-    <div className="sorter">
-      <select onChange={updateSortParam} value={value}>
-        {labels.map(([type, label]) => (
-          <option label={label} value={type} key={type}>
-            {label}
-          </option>
-        ))}
-      </select>
-    </div>
-  )
-)
+type State = {
+  menuOpen: boolean
+}
+
+const SortPicker = labels => {
+  class SortPicker extends React.Component<Props, State> {
+    constructor(props: Props) {
+      super(props)
+      this.state = {
+        menuOpen: false
+      }
+    }
+
+    toggleMenuOpen = () => {
+      const { menuOpen } = this.state
+      this.setState({ menuOpen: !menuOpen })
+    }
+
+    render() {
+      const { updateSortParam, value } = this.props
+      const { menuOpen } = this.state
+
+      return (
+        <MenuAnchor className="sorter">
+          <Menu open={menuOpen} onClose={this.toggleMenuOpen}>
+            {labels.map(([type, label]) => (
+              <MenuItem key={label} onClick={updateSortParam(type)}>
+                {label}
+              </MenuItem>
+            ))}
+          </Menu>
+          <div className="current-sort" onClick={this.toggleMenuOpen}>
+            {value}
```

Comment:
This is causing the backend value to display instead of the label (e.g. if you select "active", it displays as "hot").

Reply:
Also, while I was looking at this, I noticed the casing between the post sort labels and the comment sort labels differs (not something changed in this PR, but maybe worth fixing?)

Reply:
oh oops, lol

I'll change the casing on the labels to match too